data/app.nip05.social.md

```diff
@@ -0,0 +1,18 @@
+---
+title: App NIP-05 Social
+web: https://app.nip05.social/home
+git: 
+instances:
+tags:
+- id
```

Comment:
Not sure what's your idea of this tag. Can you please write up definitions in a separate PR for all tags used so far? You can see in WS how I did the verdicts. One file per tag. Throw tags, platforms, nips into one folder `_data/tags`.

So there are a bunch of "tools" and maybe they should be called "idServices" without the "tools" tag.

Reply:
Noted, will do next. 

idServices sounds good

Reply:
resolving this and creating the _data/tags folder in a new branch.